src/app/(Authentication)/Welcome/index.tsx

```diff
@@ -15,19 +22,29 @@ function WelcomeScreen() {
         />
 
         <View style={styles.textContainer}>
-          <Text style={styles.welcomeText}>Welcome to the Impact Fund!</Text>
+          <Text style={styles.welcomeText}>Welcome to the Impact Fund.</Text>
         </View>
         <View style={styles.buttonContainer}>
-          <StyledButton
-            text="Create an account"
-            file="/SignUp/Email"
-            color="white"
-          />
-          <StyledButton
-            text="Log in with email"
-            file="/Login/Email"
-            color="black"
-          />
+          <ButtonWhite
+            style={styles.baseButton}
+            onPress={() => router.push('/SignUp/Email')}
+          >
+            <Text style={styles.nextTextBlack}>Create an account</Text>
+            <RightArrow style={{ marginRight: 10 }} />
+          </ButtonWhite>
+
+          <View style={styles.orLine}>
+            <GreyLine style={{ margin: 5 }} />
+            <Text style={styles.orStyle}>OR</Text>
+            <GreyLine style={{ margin: 5 }} />
+          </View>
```

Comment:
I'd prefer to use a container with heigh 0 that has a bottom width. That way it expands and shrinks depending on the screen size, whereas an SVG line has the same dimensions regardless. Check the `/Components/EligibilityCard` to see how this is done.